### DISCUSSION TITLE
Consent

### DISCUSSION THREAD

**Discussion by dt-r** [hl7au/au-fhir-core#18] [AU Core R1 Resource Priority List - CLOSED]
Upvote for inclusion of a Resource Profile of [Consent](http://hl7.org/fhir/R4/consent.html) Resource in AU Core Release 1.

Comment on the profile(s) needed for Consent in AU Core.

Is it meaningful to have an AU Core Consent profile would set minimum expectations for a Consent resource to record, search, and fetch consent statements associated with a patient. 

OR 

should AU Core define one or two use case profiles e.g.
- AU Core Information Sharing Consent
- AU Core Medical Consent

**Comment by xtfer** [comment]
If by "use case profile" you mean separate profiles, then the base profile for Consent already accommodates both those use cases.

In order for a consent to be useful, and not have to be collected again, it must be validatable against a common `policy`. So, a Consent resource should try to define the possible valid `policy.authorities` and `policy.uri`'s in the Australian context. For privacy matters, these will need alignment with the Australian Privacy Act and relevant state acts. For medical consent a Common Law uri might need to be defined (?), plus state legislation. Since those imply `provisions`, it might be worth giving guidance on what those are when used.
